style.css

```diff
@@ -81,5 +86,256 @@ a:active {
   justify-content: center;
   justify-content: space-around;
   list-style: none;
-  padding-top: 10%;
+  padding-left: 102.42px;
+  padding-right: 102.42px;
+}
+
+.down-btn {
+  display: flex;
+  justify-content: center;
+  list-style: none;
+  margin-bottom: 0%;
+  padding-left: 0%;
+}
+
+.recent-work {
+  display: grid;
+  grid-template-rows: 97px 4px 474px 474px 474px 474px 474px;
+  justify-content: center;
+  place-items: center;
+  row-gap: 24px;
+}
+
+.recent-work-title {
+  text-align: center;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  margin-bottom: 0%;
+  grid-row: 1/2;
+}
+
+.section-2-green-title-line {
+  margin-top: 5%;
+  grid-row: 2/3;
+  place-items: center;
+}
+
+.w1 {
+  grid-row: 3/4;
+}
+
+.w2 {
+  grid-row: 4/5;
+}
+
+.w3 {
+  grid-row: 5/6;
+}
+
+.w4 {
+  grid-row: 6/7;
+}
+
+.w5 {
+  grid-row: 7/8;
+}
+
+.work-1-container {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  height: 474px;
+  border: 1px solid #d0d9d4;
+  background: #ebf0ee;
+  border-radius: 8px;
+  margin-top: 62px;
+}
+
+.project-image {
+  width: 100%;
+  height: 252px;
+  justify-content: flex-start;
+}
+
+.work-bottom-cont {
+  border: 1px solid #fff;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 8px;
+  justify-content: flex-end;
+}
+
+.project-title {
+  text-align: center;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #3a4a42;
+  margin-bottom: 0;
+}
+
+.Language-tags {
+  list-style: none;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  color: #3a4a42;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  padding-left: 0;
+  margin-top: 0%;
+}
+
+.lang-tag {
+  background-color: #ebf0ee;
+  padding: 8px;
+  border-radius: 4px;
+  margin: 8px;
+}
+
+.project-button {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  text-align: center;
+  padding: 12px;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  bottom: 8px;
+  background: #36b37f;
+  border-radius: 4px;
+  margin-bottom: 28px;
+  border: 0;
+}
+
+.project-button:hover {
+  background-color: #3e8e41;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  background-image: url("./Images/about-me-mobile.svg");
+  background-repeat: no-repeat;
+  margin-top: 64px;
+}
+
+.about-heading {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  order: 0;
+  margin-bottom: 24px;
+}
+
+.about-me-para {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+  margin: 0 12px 0 12px;
+  order: 1;
+}
+
+.resume-butto-cont {
+  order: 2;
+  margin-top: 24px;
+}
+
+.resume-button {
```

Comment:
Great job on implementing the button interactions as requested by the previous reviewer but kindly stick to the Figma design for the interactions as shown below 🤗

![image](https://user-images.githubusercontent.com/73714615/187034880-a65eaff6-97eb-4b2d-9d2f-e922090059c8.png)

Kindly note that the button does not change color on hover it only gets a shadow around it, and when it's pressed then it changes color, So kindly fix that 😄